cpu/stm32/cpu_init.c

```diff
@@ -370,3 +370,13 @@ void cpu_init(void)
         _wlx5xx_init_subghz_debug_pins();
     }
 }
+
+void backup_ram_init(void)
+{
+/* see reference manual "Battery backup domain" */
+#ifdef RCC_AHB1ENR_BKPSRAMEN
+    periph_clk_en(APB1, BIT_APB_PWREN);
```

Comment:
Why do we need APB1 here?

Reply:
The `st32u5` seems to be a little different then. I must take a closer look on that one. For the others APB1 is also required.
![Screenshot_20220103_142720](https://user-images.githubusercontent.com/15147337/147936031-12d289ea-0caa-4816-8dd8-9a894411f0e4.png)

My source of information which MCUs support support backup RAM was [cmsis-header-stm32](https://github.com/modm-io/cmsis-header-stm32). I just pulled and noted that `stm32h745xg`, `stm32h747xg`, `stm32u575xx`, and `stm32u585xx` have been added.

All right, I am not going  to make it a `periph_` - module in this PR.